script.js

```diff
@@ -1,4 +1,238 @@
+// Random Dice Roll Generator
+var randomDiceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var randomDiceNum = randomInteger + 1;
+  return randomDiceNum;
+};
+
+// Global Variables for Normal Dice Game
+var p1DiceOne = 0;
+var p1DiceTwo = 0;
+var p2DiceOne = 0;
+var p2DiceTwo = 0;
+var player1CombinedNum = 0;
+var player2CombinedNum = 0;
+p1DiceOne = randomDiceRoll();
+p1DiceTwo = randomDiceRoll();
+p2DiceOne = randomDiceRoll();
+p2DiceTwo = randomDiceRoll();
```

Comment:
Gameplay should happen from `main` so that it only happens on click of Submit, not on page load.